app/controllers/api/v1/mixins/validation_mixin.rb

```diff
@@ -2,12 +2,26 @@ module Api
   module V1
     module Mixins
       module ValidationMixin
-        def group_id_array_check(uuids)
-          if !uuids.kind_of?(Array)
-            invalid_parameter('Group should be an array')
-          elsif uuids.blank? || uuids.any?(&:blank?)
-            invalid_parameter('Group should not be empty')
-          end
+        def writeable_params_for_create
+          attr_list = api_doc_definition.all_attributes - api_doc_definition.read_only_attributes
+
+          params.permit(request_body_schema_keys | attr_list)
+        end
+
+        def request_body_schema_keys
+          api_doc_content.dig("components", "schemas", request_body_schema_name, "properties")
+                         .reject { |_k, v| v["readOnly"] == true }
+                         .keys
+        rescue
```

Comment:
Think we should do some sort of logging here? "Hey, some user tried to mess with attribute X!". Might also help debugging later if something either gets marked as readonly erroneously or something.

Seems like every other time we rescue we've needed the log statement, so might be a good thing to add one here. What do you think?

Reply:
Thanks for pointing this out, that rescue isn't necessary anymore since we always specify a `requestBodySchema` in our openapi.json.

The logging is also already taken care of by the common gem, it logs the "found unpermitted parameter" stuff.